source/vistas/core/graphics/overlay.py

```diff
@@ -15,7 +15,7 @@ class Overlay:
     shader = None
 
     def __init__(self, canvas):
-        self._buttons = []
```

Comment:
I was using `_buttons` here for a very specific reason; namely that `_buttons` should never be modified directly, because of other stuff that needs to happen when buttons are added or removed.

Reply:
Ah, I remember why I made this change - it was because I thought I would have to make a change inside of `GLSelectionControls`, but it ended up not happening. I'll revert it.